pkgs/applications/networking/cluster/k3s/1_24/default.nix

```diff
@@ -170,7 +170,7 @@ let
     version = k3sVersion;
 
     src = k3sRepo;
-    vendorSha256 = k3sVendorSha256;
+    vendorHash = k3sVendorSha256;
```

Comment:
It would also be best to replace `k3sVendorSha256`, which already defaults to `lib.fakeHash`, with `k3sVendorHash`.